openapi.yaml

```diff
@@ -964,7 +964,120 @@ paths:
                             schema:
                                 $ref: '#/components/schemas/Error'
                             example: 
-                                errors: ["An unexpected error occured"]    
+                                errors: ["An unexpected error occured"]
+    /projects/{project_id}/collaborators:
+        get:
+            summary: 'List project collaborators'
+            operationId: listProjectCollaborators
+            tags:
+                - Projects
+            parameters:
+              - name: project_id
+                in: path
+                description: ID of the Project
+                required: true
+                schema:
+                    type: string
+                    format: id
+              - name: page
+                in: query
+                schema:
+                    type: integer
+                    format: int32
+              - name: per_page
+                in: query
+                schema:
+                    type: integer
+                    format: int32
+            responses:
+                '200':
+                    description: 'List of project collaborators'
+                    content:
+                        application/json:
+                            schema:
+                                type: array
+                                items:
+                                    $ref: '#/components/schemas/ProjectCollaborator'
+                '400':
+                    description: 'Bad Request'
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/Error'
+                            example: 
+                                errors: ["Bad Request"]
+                '404':
+                    description: 'Not found'
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/Error'
+                            example: 
+                                errors: ["Record not found"]
+                '500':
+                    description: Unexpected error
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/Error'
+                            example: 
+                                errors: ["An unexpected error occured"]
+    /projects/{project_id}/collaborators/invite:         
```

Comment:
Is there a reason we don't go with just `/projects/{project_id}/collaborators` to be more CRUD like? That or we could go a step further if we don't think it's intent revealing and do POST `projects/{project_id}/invited_collaborators`.

Reply:
I'm more in favour of doing something like [what Jeff suggested](https://github.com/CompanyCam/openapi-spec/pull/10/files#r1016871610)

Reply:
Yeah, we thought about this, but I don't love it because we're not actually creating a collaborator here, just creating the invite link. I'm down with Jeff's suggestion though. I think that makes a ton of sense.